config/routes.rb

```diff
@@ -130,7 +130,9 @@
 
     scope :facilities, module: 'facilities' do
       resources :va, only: %i[index show], defaults: { format: :json }
+      resources :ccp, only: %i[show], defaults: { format: :json }
       get 'suggested', to: 'va#suggested'
+      get 'specialties', to: 'ccp#specialties'
```

Comment:
shouldn't this be nested under ccp?

Reply:
I just decided to match the way the "suggested" endpoint is set up, living in a nested controller, but routed at the level above. it seems to work well enough this way.